5_People/benefits.md

```diff
@@ -118,7 +118,7 @@ We are from different parts of the World, and we have different customs when it
 
 ## Sick Leave
 
-Everyone has the right to sick leave on days when they don't feel capable of working. A day of sick leave counts as 6 hours of work done and will be automatically added to your "worked hours".
+Everyone has the right to sick leave on days when they don't feel capable of working. A day of sick leave counts as 6 hours of work done and will be automatically added to your "worked hours". Sick leave can and should be used when taking care of an ill child or spouse.
 
-Sick leave can and should be used when taking care of an ill child or spouse.
+There is a total of 20 sick days per year available before unpaid vacation is calculated. For more than five consequential sick days, a doctor's note is required.
```

Comment:
consequential -> maybe the word is `consecutive`?

Reply:
Yes, that's the correct word.